Feat cookie alert

OPerel opened the pull request:
# Description
Added `<CookieAlert />` component that pops up the first time a user visits the website, or if the user did not accept cookie use, every five days.
The cookie alert component is used in `_app.tsx`, so it will pop up even if the user visits a page other than `/`.
Few other things to note:
- The reference example didn't have a close or dismiss button, so I added a button reading 'Dismiss' in the bottom right corner of the alert. It closes the alert and sets the current time + 5 days in `localStorage`.
- The rest of the text is copied exactly as it is in the example (including the Unicode emoji).
- The alert pop ups in the left bottom corner of the screen (covers the entire screen width on smaller than the `md` breakpoint).

This is how it looks like:
![image](https://user-images.githubusercontent.com/42510547/137617423-63f88e4a-d6d6-498e-8442-a02d20dc8e26.png)

Fixes #79 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
I tested code manually in dev mode.
**I was unable to run the production build locally.** This is true also without the new cookie alert component. So I didn't test locally in production mode.
Not sure why the production build failed to run. Both the `build` and `start` commands gave no warnings or errors. 
All I got was `An unexpected error has occurred.` message in the browser window:
![image](https://user-images.githubusercontent.com/42510547/137616848-c04b6b54-644d-4b23-af58-6e9c1d5b31cc.png)

And `TypeError: Cannot read properties of undefined (reading 'map')` in the console:
![image](https://user-images.githubusercontent.com/42510547/137616898-5e57d414-49fb-46d4-bf5c-58765077a3b5.png)

The `undefined` refers to the `socialMedia` variable in `socialbar.tsx`, so I'm guessing the problem is due to DB connection, but this is as far as I could figure it out.
I have to admit I don't understand the first point described in the PR process in the contributing guidelines, so maybe I'm doing something wrong... 🤷‍♂️ 
The only thing I was able to see when running the production build was the cookie alert popping up...

**Test Configuration**:
I'm on Windows 10
Node version is 14.16.0.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings (at least in dev mode)
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

harshgoel05 commented:
![Screenshot_2021-10-19-13-47-50-01](https://user-images.githubusercontent.com/48649088/137870872-a9d3b1ca-44a7-4139-a2bc-4d4c4986cb8a.jpg)

Also the z index issue on phone.
The paragraph must be smaller than heading

OPerel commented:
Hey @harshgoel05 
I fixed the font size and z-index.
Here are full page screenshots.
Mobile:
![image](https://user-images.githubusercontent.com/42510547/137903113-d0aace84-7854-4836-931b-72f5fe3e8f62.png)

Desktop:
![image](https://user-images.githubusercontent.com/42510547/137902460-41cccb44-e1b4-44c1-8424-3c392c9a274c.png)

I reverted the changes in `yarn.lock`.
I followed the discussion in #91 and now my production build is working. 

I see vercel failed to deploy the preview. Please let me know if there's something I can do to fix it.
✌️

harshgoel05 commented:
Thanks for the PR! This looks good to me! @OPerel
